cluster-api-gcp/generate-yaml.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+set -e
+
+GCLOUD_PROJECT=$(gcloud config get-value project)
+
+TEMPLATE_FILE=machines.yaml.template
+GENERATED_FILE=machines.yaml
+
+if [ -f $GENERATED_FILE ]; then
+  echo File $GENERATED_FILE already exists. Delete it manually before running this script.
+  exit 1
```

Comment:
I personally would prefer it to overwrite or else every iteration I will need to explicitly delete the old version. Perhaps a flag to control whether it will overwrite or not?

Reply:
The reason I opted for not overwriting is to avoid losing any configuration changes made during development. 

However, I agree that one might prefer to overwrite always. Added flag.